cmd/px-node-wiper/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM portworx/px-enterprise:1.3.0-rc4
```

Comment:
this is need since pxctl isn't a static binary and has dependencies to run which are present in the px-enterprise container. Once we release 1.3.0, the image tag will be updated accordingly.